projects/js-themes-toolkit/packages/liferay-theme-tasks/theme/__tests__/index.test.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const {r2} = require('liferay-theme-tasks');
+
+describe('index.js', () => {
+	it('exposes the r2 API from the "main" package resource', () => {
+		expect(require('liferay-theme-tasks')).toBe(require('..'));
+
+		expect(Object.keys(r2).sort()).toEqual(['exec', 'swap', 'valueMap']);
+	});
+
+	describe('r2.swap()', () => {
+		const {swap} = r2;
+
+		it('swap text alignment', () => {
+			expect(swap('p{text-align:right;}')).toEqual(
+				'p{text-align:left;}',
+				'text-align: left => text-align: right'
+			);
+			expect(swap('p{text-align:left;}')).toEqual(
+				'p{text-align:right;}',
+				'text-align: right => text-align: left'
+			);
```

Comment:
Finally, a quick test to show you can use it to actually do something — this is a small subset of [the tests that over in the actual module itself](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/js-themes-toolkit/packages/liferay-theme-tasks/lib/r2/liferay-r2/__tests__/index.test.js).